.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] Bug description"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Input:
+```java
+/*Code being processed by spoon and leading to the bug.*/
+```
+
+Processing with Spoon:
+```java
+/*Code calling spoon on the input and triggering the bug.*/
+```
+
+Output:
+```java
+/*Output code or stacktrace if unexpected Exception is raised*/
+```
+
+Note that a pull request containing a test demonstrating the bug is even more welcome than an issue!
+
+**JDK and Spoon version used**
```

Comment:
Add OS (Windows does funny things) and perhaps provide some more obvious "help"?

```suggestion
**Operating system, JDK and Spoon version used**

* OS:
* JDK:
* Spoon version:
```